elastic/kibana
# [Fleet] Package telemetry

## Summary

Added telemetry for package installs and updates https://github.com/elastic/kibana/issues/111136

To test:
- install any package from registry or upload
- wait 30s and check debug logs to see the telemetry event is send with `package-install` data
- simulate an error and see the errorMessage being sent in telemetry event as well (e.g. try installing outdated package)
